src/HeaderSort/components/HeaderSortContainer.ts

```diff
@@ -88,7 +87,7 @@ export default class HeaderSortContainer extends Component<ContainerProps, Conta
         return null;
     }
 
-    private connectToListView() {
+    private connectToListView = () => {
```

Comment:
Please avoid arrow functions in class properties.
Please refer to https://medium.com/@charpeni/arrow-functions-in-class-properties-might-not-be-as-great-as-we-think-3b3551c440b1